src/github.com/matrix-org/dendrite/syncapi/storage/syncserver.go

```diff
@@ -211,7 +212,7 @@ func (d *SyncServerDatabase) syncStreamPositionTx(
 // IncrementalSync returns all the data needed in order to create an incremental sync response.
```

Comment:
this is a public function, so definitely needs documentation of what `device` does.